fontbe/src/features.rs

```diff
@@ -182,10 +182,15 @@ pub(crate) fn resolve_variable_metric<'a>(
         // Array of region axis coordinates records, in the order of axes given in the 'fvar' table.
         let mut region_axes = Vec::with_capacity(static_metadata.axes.len());
         for axis in static_metadata.axes.iter() {
-            let Some(tent) = region.get(&axis.tag) else {
-                todo!("FIXME: add this error conversion!")
+            let coords = match region.get(&axis.tag) {
+                Some(tent) => tent.to_region_axis_coords(),
+                None => {
+                    // it would be nice if this was all validated before now
+                    log::warn!("unknown axis '{}' in region", axis.tag);
```

Comment:
why isn't this an error?

Reply:
This felt like an expedient compromise:

- adding a variant to the `DeltaError` that we're returning doesn't seem right, since this is technically unrelated
- adding a new error type just for this function feels like overkill, especially because:
- how would this error condition ever occur? The location here is the location for a given master, which was retrieved (afaict) from the static metadata. Presumably when we are generating the IR we ensure that the locations for various masters are defined in a common axis-space. The locations are all reused for all sets of deltas. At no point do we conjure up new locations.

realistically, it feels like this should maybe be a fatal error, since it probably represents a major inconsistency in our inputs?

Reply:
I think you can unwrap here

Reply:
+1, unwrap (if it's definitely meant to be impossible) or error, a warning log for a condition that should never occur doesn't seem right.